tools/api_generator/dto_codegen.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+data_output_path=../../lib/api/data
+umbrella_file_name=umbrella.dart
+
+### Generate swagger configuration
+for i in $(find ../../../flutter-template-swagger/**/api.yaml); do
+  ./apple_silicon_generator generate "$i" -c dto_config.yaml -n "$(basename $(dirname "$i"))" -r
+done
+
+### Move to data api folder
+cd ${data_output_path}
+
+### Generate all *.g.dart files
+fvm flutter clean && fvm flutter pub get && fvm flutter pub run build_runner build --delete-conflicting-outputs
+
+### Remove previous generated export file
+rm -f ${umbrella_file_name}
+
+### Iterate over all files, except *.g.dart files
+rm -rf build
+fileNames=()
+filter=".g.dart"
+for file in *
+do
+  nameAndExt=${file##*/}
+    if [[ "$nameAndExt" != *"$filter"* ]]; then
+        fileNames+=("$nameAndExt")
+    fi
+done
+
+### Create new umbrella file
+for value in "${fileNames[@]}"
+do
+  name=${value%.*}
+  echo "export '$name.dart';" >> "$umbrella_file_name"
+done
+
+### Format all generated files
+fvm dart format -l 100 lib packages test
```

Comment:
```suggestion
### TODO(init-project): change to your project line length and desired folders for formatting
fvm dart format -l 120 lib packages test
```

Reply:
Right now, in our project, the line length is 100 everywhere, can we leave 100 here?

Reply:
We will change to 120 later. I will write it do to not to forget. Yep, let's leave it 100 for now.